src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="1.0.1" />
+    <PackageReference Include="OpenTelemetry.Api" Version="1.0.1" />
```

Comment:
`EntityFrameworkCore.Tests` project depends on Sdk for some of the unit tests. Do we want to add the SDK dependency in `OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore.Tests.csproj`?